Lib/asyncio/events.py

```diff
@@ -155,6 +155,10 @@ def close(self):
         """Coroutine to wait until service is closed."""
         return NotImplemented
 
+    def get_loop(self):
+        """ Get the event loop"""
```

Comment:
`""" Get the event loop"""` -> `"""Get the event loop the Server object is attached to."""`